### PR TITLE
feat: Add MultiPeopleSelectField form field

### PR DESCRIPTION
**Pull request by Rockyy174**
I've added a new form field: `MultiPeopleSelectField`

### CODE REVIEW

File: assets/js/components/Forms/MultiPeopleSelectField.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import Avatar from "../Avatar";
```
Comment (shiroyasha) [inline assets/js/components/Forms/MultiPeopleSelectField.tsx:3]: Use absolute path to make it easier to move the file around if necessary `@/components/Avatar.`

File: assets/js/components/Forms/FormState.tsx [hunk 2]
```diff
@@ -15,8 +17,8 @@ export interface FormState<FieldTypes extends MapOfFields> {
 
 export type Field<T> = {
   value: T | null | undefined;
-  setValue: (value: T | null | undefined) => void;
-  initial: T | null | undefined;
+  setValue: Dispatch<SetStateAction<T | undefined>>;
+  initial?: T | null | undefined;
```
Comment (shiroyasha) [inline assets/js/components/Forms/FormState.tsx:21]: My TS typefoo is a bit weak. What is the difference between `initial?: ` and `initial: ... | undefined` and `initial?: ... | undefined`.

**Comment by shiroyasha** [comment]
Everything looks good. I've added some nitpick comments.

File: assets/js/components/Forms/FormState.tsx (hunk 2 above)
Comment (Rockyy174) [reply assets/js/components/Forms/FormState.tsx:21]: Good question. When I added `?`, I didn't notice that `undefined` was already there. 

I also thought that `?` and `undefined` were the same thing, but without `?`, TS complains.
It seems there is a difference.

**Comment by Rockyy174** [comment]
I've updated the import and double checked `type Field<T>` (it won't work without `?`).
This PR should be ready then.

File: assets/js/components/Forms/FormState.tsx (hunk 2 above)
Comment (shiroyasha) [reply assets/js/components/Forms/FormState.tsx:21]: We are still young typescript jedies, it seems that both you and me don't yet fully understand every typescript typing option 👦🏻

**Pull request merged**